contracts/CarbonVoteX.sol

```diff
@@ -0,0 +1,207 @@
+pragma solidity ^0.4.23;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
```

Comment:
remove the extra blank line

Reply:
I think two blank lines between import and contract is okay, which follows python coding styles.

Reply:
ok, ignore this one